.eslintrc

```diff
@@ -0,0 +1,17 @@
+{
+  "parser": "@typescript-eslint/parser",
+  "extends":  [
+    "airbnb-base",
+    "plugin:@typescript-eslint/recommended",
+    "prettier",
+    "prettier/@typescript-eslint"
+  ],
+  "plugins": ["@typescript-eslint", "prettier"],
+  "rules": {
+    "@typescript-eslint/explicit-function-return-type": "off",
+    "@typescript-eslint/explicit-member-accessibility": "off",
+    "import/no-unresolved": "off",
+    "import/extensions": "off",
```

Comment:
I know that @typescript-eslint’s plugin is supposed to be able to figure the import problems, but for whatever reason I end up having to override these every time. TypeScript catches these.